lighthouse-core/test/lib/network-request-test.js

```diff
@@ -176,5 +176,64 @@ describe('NetworkRequest', () => {
       expect(record).toMatchObject(req);
       expect(record.lrStatistics).toStrictEqual(undefined);
     });
+
+    it('Handles negative timing data', function() {
```

Comment:
maybe add another similar test that shows that it works if a header is missing? (since it'll default to 0 in that case, as long as the numbers still add up it should work)